### PR TITLE
Update ko.json

### PR DESCRIPTION
**Pull request by Flamenk**
## What is the purpose of the change
Corrected typos and incorrect translations in Korean
ex) Pool -> 'Swimming pool' to 'Liquidity pool' in Korean

& I found that there is no Japanese pack in the localization package, could I create a JP version?
<!-- > Add a description of the overall background and high level changes that this PR introduces

### CODE REVIEW

**Review by dogemos — APPROVED** [review]

**Comment by dogemos** [comment]
LGTM

**Review by gnujoow — APPROVED** [review]
LGTM
way better than existing transalation

File: packages/web/localizations/ko.json [hunk 1]
```diff
@@ -289,7 +289,7 @@
       "poweredBy": "제공:",
       "proceed": "진행하다",
       "depositExpiration": "이 입금 주소는 다음 날짜에 만료됩니다.",
-      "undergoAudit": "감사 중(2024년 1월 완료 예정)",
+      "undergoAudit": "오딧 진행중",
```
Comment (gnujoow) [inline packages/web/localizations/ko.json:292]:
```suggestion
      "undergoAudit": "감사 진행중",
```
I believe that "감사" deliver its meaning more accuratly.

File: packages/web/localizations/ko.json [hunk 2]
```diff
@@ -432,14 +432,14 @@
...
     "signupDummyHistoryMessage2": "Osmosis로 자금을 이체하는 방법 알아보기",
     "signupDummyHistoryTitle3": "거래를 시작하는 방법",
-    "signupDummyHistoryMessage3": "이 빠른 자습서를 통해 몇 분 안에 거래할 수 있습니다.",
+    "signupDummyHistoryMessage3": "이 빠른 가이드를 통해 몇 분 안에 거래할 수 있습니다.",
     "outboundTransferHistoryTitle": "Osmosis 아웃바운드 전송",
     "outboundTransferHistoryMessage": "양",
     "poolExitedHistoryTitle": "Osmosis 풀 종료됨",
     "poolExitedHistoryMessage": "풀 ID",
-    "poolJoinedHistoryTitle": "삼투 풀 합류",
+    "poolJoinedHistoryTitle": "Osmosis 풀 합류",
```
Comment (gnujoow) [inline packages/web/localizations/ko.json:440]:
```suggestion
    "poolJoinedHistoryTitle": "오스모시스 풀 합류",
```
match expression "삼투" to "오스모시스". only this on is written in English.

**Review by gnujoow — COMMENTED** [review]
Small change suggestion.

**Comment by MaxMillington** [comment]
@Flamenk Thanks for the contribution! A Japanese file would be great!

**Pull request merged**